library/lightwave

```diff
@@ -0,0 +1,9 @@
+# VMware Lightwave
+Maintainers: Sriram Nambakam <snambakam@vmware.com> (@snambakam), Lightwave Checkins <lightwave-checkins@vmware.com> (@gerrit-lightwave)
+GitRepo: https://github.com/vmware/lightwave.git
+ 
```

Comment:
There's a single "space" (` `) on this line which is turning the value of `GitRepo` into `https://github.com/vmware/lightwave.git\n` instead of separating the two entries.